dev_requirements.txt

```diff
@@ -0,0 +1,3 @@
+coverage
+mock
+sphinx;platform_system=="Windows"
```

Comment:
Why is this conditional here? Sphinx is required always for building the docs, not just in windows.